IETF.js

```diff
@@ -0,0 +1,217 @@
+{
+	"translatorID": "ac4b1d6a-6e52-42e2-9634-ebbe1e96b9ad",
+	"label": "IETF",
+	"creator": "Félix Brezo (@febrezo)",
+	"target": "^https?://tools.ietf.org/(html|pdf|rfc)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-08-02 02:31:35"
+}
+
+/*
+	IETF Translator
+	Copyright (C) 2020 Félix Brezo (@febrezo), felixbrezo@disroot.org
+	
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	
+	You should have received a copy of the Affero GNU General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+function detectWeb(doc, url) {
+	return "report";
+}
+
+function doWeb(doc, url) {
+	var resourceType = detectWeb(doc, url);
+	var newItem = new Zotero.Item(resourceType);
+	
+	var targetUri;
+	if (url.indexOf("/rfc/") > -1) {
```

Comment:
`.includes()`, not `indexOf()`, and same below, but it'd be better to just replace this whole set of conditions with a simple regexp that just extracts the RFC number, and then build a new URL with that.